app/Database/CourseInsertion.hs

```diff
@@ -46,17 +46,17 @@ saveGraphJSON jsonStr nameStr = do
 -- Get Key of correspondig record in Distribution column
 getDistributionKey :: Maybe T.Text -> SqlPersistM (Maybe (Key Distribution))
 getDistributionKey Nothing = return Nothing
-getDistributionKey (Just description) = do
-    keyListDistribution :: [Key Distribution] <- selectKeysList [ DistributionDescription ==. (T.unpack description) ] []
+getDistributionKey (Just description_) = do
+    keyListDistribution :: [Key Distribution] <- selectKeysList [ DistributionDescription ==. (T.unpack description_) ] []
     -- option: keyListDistribution :: [DistributionId] <- selectKeysList [ DistributionDescription `contains'` description] []
     return $ case keyListDistribution of
         [] -> Nothing
         _ -> Just (head keyListDistribution)
 
 getBreadthKey :: Maybe T.Text -> SqlPersistM (Maybe (Key Breadth))
 getBreadthKey Nothing = return Nothing
-getBreadthKey (Just description) = do
-    keyListBreadth :: [Key Breadth] <- selectKeysList [ BreadthDescription ==. (T.unpack description) ] []
+getBreadthKey (Just description__) = do
```

Comment:
The `description_` above is a parameter of `getDistributionKey`, so it's local to that function. You can use the same parameter name here in `getBreadthKey` without getting the shadowing warning.